src/test/resources/failing_resource/FileEmbargo.txt

```diff
@@ -0,0 +1,6 @@
+   concat                |                             text_value                              | start_date
```

Comment:
Måtte legge til FileEmbargo ett par plasser for at de gamle testene skulle kjøre.